ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/TopicStream.java

```diff
@@ -179,7 +180,9 @@ public String print(final ConsumerRecord<String, Bytes> record)
 
             objectNode.put(SchemaUtil.ROWTIME_NAME.name(), record.timestamp());
             objectNode.put(SchemaUtil.ROWKEY_NAME.name(), key);
-            objectNode.setAll((ObjectNode) jsonNode);
+            if (jsonNode != null) {
```

Comment:
This fix isn't necessary since the above addition (on line 71) takes care of it, but I figure the redundancy can't hurt since we don't want to hit an NPE if something else we don't know about slips through?

Reply:
I prefer omitting this fix because jsonNode being null could be a symptom for a different issue (assuming we expect it to be non-null)

Reply:
Done.